sdk/purview/purview-sharing-rest/test/public/shareResourcesTest.spec.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { isUnexpected, PurviewSharingClient, ShareResourceListOutput } from "../../src";
+import { Recorder } from "@azure-tools/test-recorder";
+import { assert } from "chai";
+import { createClient, createRecorder } from "./utils/recordedClient";
+import { Context } from "mocha";
+
+describe("Sent Shares Operations", () => {
+  let recorder: Recorder;
+  let client: PurviewSharingClient;
+
+  beforeEach(async function (this: Context) {
+    recorder = await createRecorder(this);
+    client = createClient(recorder);
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  it("List all Share Resources", async function () {
+    const response = await client.path("/shareResources").get();
```

Comment:
Like sample code, could we leverage `paginate` helper function to get all items?